specs/integration/api/graphql/conversations.query.spec.ts

```diff
@@ -107,6 +112,55 @@ describe('GraphQL Conversations Query', () => {
       expect(data.conversations[0].customer.conversations[0].id).to.eq(conversation.id)
     })
 
+    it('returns the messages ordered by most recent first by default', async () => {
```

Comment:
```javascript
const oldestMessage = message

const now = new Date();

timekeeper.travel(addDays(now, 3));

const newerMessage = await factories.messageFactory.create({
  conversationId: conversation.id,
  authorId: user.id,
  authorType: AuthorType.STAFF
})

timekeeper.travel(addDays(now, 5));

const newestMessage = await factories.messageFactory.create({
  conversationId: conversation.id,
  authorId: user.id,
  authorType: AuthorType.STAFF
})

// ...

expect(
  _.map(data.conversations[0].messages, 'id')
).to.deep.equal([
  newestMessage.id,
  newerMessage.id,
  oldestMessage.id,
])
```
Basically reordering the initial setup part of the test from old, newer, newest. Instead of oldest, newest, newer.
I also found it easier to reason about.